src/components/Settings.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <v-card max-width="20em" max-height="100%" style="overflow: auto;">
+  <v-card max-width="30em" max-height="100%" style="overflow: hidden;">
```

Comment:
Is there a reason you changed overflow to hidden? Now you can't scroll on short screens.

Reply:
Oh, whoops. I thought it was changing some of the layout... I'll test it.

Reply:
Didn't seem to affect anything... it should work now as a result.